cockatrice/src/gameselector.cpp

```diff
@@ -74,18 +74,16 @@ GameSelector::GameSelector(AbstractClient *_client,
     connect(filterButton, SIGNAL(clicked()), this, SLOT(actSetFilter()));
     clearFilterButton = new QPushButton;
     clearFilterButton->setIcon(QPixmap("theme:icons/clearsearch"));
-    if (showFilters && gameListProxyModel->areFilterParametersSetToDefaults()) {
-        clearFilterButton->setEnabled(false);
-    } else {
-        clearFilterButton->setEnabled(true);
-    }
+    const bool filtersSetToDefault = showFilters && gameListProxyModel->areFilterParametersSetToDefaults();
```

Comment:
this const here isn't going to be doing anything

Reply:
Fixed